src/ipyforcegraph/graphs.py

```diff
@@ -66,6 +67,35 @@ def reheat(self) -> None:
         """Send the reheat command to restart the force simulation"""
         self.send({"action": "reheat"})
 
+    @T.validate("behaviors")
+    def _check_behavior_order(self, proposal: T.Bunch) -> Tuple[Behavior, ...]:
+        """Ensure behaviors are not unwittingly being put in the wrong order."""
+        behaviors: Tuple[Behavior, ...] = proposal.value
+
+        # TODO: Investigate making ordering a Behavior class variable
```

Comment:
yerp, rather than adding this business, would see adding a `rank` to the behaviors, which get sorted (ha) out on the frontend, with some sensible defaults... warnings and complex validation methods can still get swallowed when things are done "live,"

Reply:
Made an issue #76, unless we want to fix it in this PR